html/js/tree/ISet.js

```diff
@@ -0,0 +1,151 @@
+GTE.TREE = (function (parentModule) {
+    "use strict";
+
+    /**
+    * Creates a new ISet.
+    * @class
+    * @param {ISet} [parent] Parent ISet. If null, this is root.
+    * @param  {Number} [numOfNodes] Number of nodes of this new information set. If null 1
```

Comment:
Or, is this maybe a standard with constructor parameters? (I don;t remember seeing it before) If so leave them :)